doc/tutorials/11-ferrofluid/11-ferrofluid_part1.ipynb

```diff
@@ -88,13 +88,13 @@
     "\n",
```

Comment:
<p><strong>an</strong> animation</p><p> </p><p>remove hashtag</p><p> </p><p>xdata and ydata should be named x_data and y_data</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/espressomd/espresso/pull/3919/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
<p>Which hashtag are you referring to?</p>

Reply:
it was probably an artefact of the reviewNB app, nevermind